home/jobs/Trigger/config.xml

```diff
@@ -13,11 +13,16 @@
     </org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty>
   </properties>
   <definition class="org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition" plugin="workflow-cps@2.36.1">
-    <script>build job: &apos;BIOFORMATS-push&apos;
+    <script>
+build job: &apos;MANAGEMENT_TOOLS-merge&apos;
```

Comment:
The corresponding job was not added. Is that a requirement for this addition?

Reply:
Let me check. Likely this was only needed for the coupled build, which comes next.